README.md

```diff
@@ -11,54 +11,60 @@ Please direct any queries & feedback to [the developers](https://github.com/saml
 ## Examples of topics the bot can help with
 
 General election help:
+
 - what is an election/ how does it work
 - how to nominate (myself/someone/others)
 - how to vote
 - who should I vote for/ how to decide who to vote for
 - how is the candidate score calculated
-- **what is my candidate score** *(calculates if ownself is eligible for nomination)*
+- **what is my candidate score** _(calculates if ownself is eligible for nomination)_
 - what are the moderation badges
 - what are the participation badges
 - what are the editing badges
-- what are the required badges *(SO-only)*
+- what are the required badges _(SO-only)_
 
 Current election info:
+
 - election schedule
 - what is the election status
 - when is the election starting/ when is the next phase
 - when is the election ending
 - who are the candidates/nominees
 - who are the winners/new moderators
+- how many positions are elected
```

Comment:
```
How many moderators will be elected?
How many positions are available?
```

Reply:
> need it to say "will be" if we are in the past

Yeah, `will` is already scheduled 😇 I just wanted to give it a shot as a proof of concept